app/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        fields = '__all__'
```

Comment:
don't use all mention the fields so we can use take data from different tables too